samples/bluetooth/iso_broadcast/README.rst

```diff
@@ -21,7 +21,8 @@ Building and Running
 ********************
 
 This sample can be found under :zephyr_file:`samples/bluetooth/iso_broadcast` in
-the Zephyr tree.
+the Zephyr tree. Use `-DOVERLAY_CONFIG=overlay-bt_ll_sw_split.conf` to enable
+required ISO feature support in Zephyr Bluetooth Controller on supported boards.
```

Comment:
This may be nitpicking: But how does a user determiner which board is supported? The `platform_allow` can no longer be used, as that only contains a single board (`nrf52dk_nrf52832`), but I guess this also works for the nrf52840, right?

Reply:
Good question again, myself do not know how to figure out which samples would work on which boards! I will ask the experts... and get back

Reply:
@carlescufi said 
> this is all in flux in the upstream project, there are no clear definitions

I will leave you question on, which board is supported, out of the scope of this PR.